Geography docs (#344)
betterlytics/betterlytics: docs/src/content/dashboard/geography.mdx

---
title: "Geography Analytics - Betterlytics"

> description: "Visualize your website visitors' geographic distribution on an interactive world map to understand your global audience and optimize for regional markets."

Reviewer: Consider splitting this into two sentences to make it easier to read and understand.

---

import { Callout } from "nextra/components";

# World Map Analytics

> The World Map provides a visual representation of where your website visitors are located around the world. This interactive map helps you understand your global audience distribution, identify key markets, and make data-driven decisions about international expansion, content localization, and regional marketing strategies.

Reviewer: ```suggestion
The World Map provides a visual representation of where your website's visitors are located around the world. This interactive map helps you understand your global audience distribution, identify key markets, and empower data-driven decisions about international expansion, content localization, and regional marketing strategies.
```

* The possessive form "website's" indicates that the visitors belong to the website.
* `make data-driven decisions` feels a bit flat, so I changed 'make' to 'empower'

## Accessing the World Map

> Navigate to **Dashboard > Geography** to view your world map analytics data.

Reviewer: ```suggestion
The world map analytics are available directly from the Dashboard Overview, or you can view more detailed insights by going to Dashboard > Geography.
```

* Mention that it's also on the dashboard overview
* 'Navigate' feels a bit like a command - might be better with a more subtle wording

Author: I kept the navigate to be consistent with the other pages. Lets go through all docs at some point and ensure consistency between them - once they're all merged :)

I added a mentioning of the world map on the dashboard overview

## What the World Map Shows

> The world map displays **unique visitors** (not page views) by country, giving you insight into your actual audience reach rather than just activity volume. Each country is color-coded based on the number of unique visitors from that location during your selected time period.

Reviewer: ```suggestion
The world map visualizes unique visitors (rather than page views) by country, offering insights into your actual audience reach instead of just activity volume. Each country is color-coded based on the number of unique visitors during the selected time period.
```

* "Displays" vs. "Visualizes" – "Visualizes" may better capture the idea that it's a map or graphical representation - I personally prefer "depicts," as it conveys a straightforward visual representation, but I understand that it might feel a bit too artistic for general documentation. "Visualizes" or "displays" might be more fitting for clarity and accessibility in a technical context.
* "Giving you insight into" vs. "Offering insights into" – "Offering insights into" sounds a bit more formal and polished.
* "During your selected time period" – I changed “your” to “the selected” for a more neutral tone and to avoid ambiguity about whether it's the user's time period or a pre-set one

Author: Rewritten 👍

<div style={{ textAlign: "center", margin: "20px 0" }}>
  ![World Map](/images/dashboard/world-map.png)
</div>

<Callout type="info">

> The map shows **unique visitors** - meaning each person is counted only once per country, regardless of how many pages they visited. This gives you a true picture of your geographic audience reach.

Reviewer: ```suggestion
The map shows **unique visitors** - each person is counted once per country, regardless of page visits. This gives you a true picture of your geographic audience reach.
```

> See [Metrics Glossary](/dashboard/metrics-glossary) for more information on how unique visitors are calculated.

Reviewer: We mention `unique visitors` so many times, that it should probably be emphasized in the text :D

</Callout>

## Map Features

### Interactive Country Data

**Country Visualization:**
- **Color-coded countries** - Each country is shaded based on visitor volume
- **Blue gradient scale** - Lighter blue indicates higher visitor counts
- **Gray countries** - Locations with no visitors appear in gray
- **Hover information** - Click any country to see exact visitor numbers

### Map Controls

**Navigation:**
- **Zoom controls** - Use the + and - buttons to zoom in and out or use the scroll wheel on your mouse
- **Pan and drag** - Click and drag to move around the map
- **Bounded view** - The map prevents excessive panning to maintain usability

### Legend

**Understanding the Scale:**
- **Color legend** - Located in the bottom-right corner
- **Gradient scale** - Shows the range from 0 visitors to your maximum
- **Numeric range** - Displays actual visitor numbers for context

## Data Accuracy and Sources

### Geographic Detection

**How Location is Determined:**
- **IP-based geolocation** - Visitor location is determined from their IP address
- **Country-level precision** - Data is aggregated at the country level

> - **GeoIP database** - Uses MaxMind GeoLite2 database for accurate location mapping

Reviewer: ```suggestion
- **GeoIP database** - Uses [MaxMind GeoLite2](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data/) database for accurate location mapping
```

Add hyperlink to maxmind
